charts/drax/charts/dashboard/templates/configmap-core-ui.yaml

```diff
@@ -14,5 +14,5 @@ name: {{ include "accelleran.common.fullname" (dict "top" $) }}-core-ui
 data:
   config.js: |-
     window.GRAFANA_PORT = "{{ tpl $.Values.config.grafanaPort $ }}";
```

Comment:
I think we need to indicate this as required as well? At least in combination with the `0` value that's what I would expect

Reply:
The impact of this not being properly configured is solely that the link to Grafana in the Dashboard doesn't work. Given that this is quite minor, doesn't it make more sense to allow for the Dashboard to be deployed in a scenario where Grafana isn't actually available? Marking it as required would essentially just be asking the user to input a garbage random value in order to proceed with installation.